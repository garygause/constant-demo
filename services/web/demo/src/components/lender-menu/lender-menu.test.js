import React from "react";
import { shallow } from "enzyme";
import LenderMenu from "./lender-menu.component";

import Enzyme from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({ adapter: new Adapter() });

let wrapper;
beforeEach(() => {
  const mockProps = {
    lenders: [
      {
        id: 1,
        name: "test",
        address: "123",
        city: "Seattle",
        state: "WA",
        zip: "98115",
      },
      {
        id: 2,
        name: "test",
        address: "123",
        city: "Seattle",
        state: "WA",
        zip: "98115",
      },
    ],
  };

  wrapper = shallow(<LenderMenu {...mockProps} />);
});

it("should render LenderMenu component", () => {
  expect(wrapper).toMatchSnapshot();
});
