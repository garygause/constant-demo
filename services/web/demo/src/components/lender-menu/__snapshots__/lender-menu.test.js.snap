// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LenderDetails component 1`] = `ShallowWrapper {}`;

exports[`should render LenderMenu component 1`] = `ShallowWrapper {}`;
